Hello_World.js

```diff
@@ -0,0 +1,2 @@
+//Program to print hello world
```

Comment:
file name shoud only have small lettters

Reply:
where is the program no in the file

Reply:
added the renamed file.